client/src/routes.js

```diff
@@ -14,16 +14,32 @@ import Callback from "./components/Callback/callback";
 //History
 import history from "./history";
 
+import { Provider } from "react-redux";
+import { createStore, applyMiddleware, compose } from "redux";
+import thunk from "redux-thunk";
+import logger from "redux-logger";
+
+import rootReducer from "./store/reducers";
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
I've had issues with this going into production. If the user doesn't have chrome redux tools, the page won't load.